letta/settings.py

```diff
@@ -11,7 +11,7 @@ class ModelSettings(BaseSettings):
 
     # openai
     openai_api_key: Optional[str] = None
-    # TODO: provide overriding BASE_URL?
+    openai_base_url: Optional[str] = "https://api.openai.com/v1"
```

Comment:
The default for the base url is now pulled up here.

Reply:
also FYI I think this env var is usually `OPENAI_API_BASE`, not `OPENAI_API_BASE_URL`, but I'm not sure how to change this in the pydantic settings field other than renaming the var to `openai_base`?

I guess it's also fine for us to use a different env var? idk

eg https://github.com/langchain-ai/langchain/issues/10027

Reply:
oh yeah you can just change it to be `openai_api_base` here - and we can be consistent with the other providers

Reply:
done